src/ThreeEditor/components/Sidebar/propeteries/category/BeamConfiguration.tsx

```diff
@@ -131,6 +131,68 @@ function BeamSigmaField(props: { beam: Beam; onChange: (value: Beam['sigma']) =>
 	);
 }
 
+
+function BeamSadField(props: { beam: Beam; onChange: (value: Beam['sad']) => void }) {
+	const configuration = {
+		[SAD_TYPE.Rectangular]: {
+			X: {
+				text: 'sad in X'
+			},
+			Y: {
+				text: 'sad in Y'
```

Comment:
```suggestion
				text: 'SAD Y'
```

Reply:
Should be OK now.
P.S.
Sorry, I uses plugin in VScode and it didn't show suggestions, so I changed it manually.